lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -31,8 +33,9 @@ import GroupConfigSelector from './GroupConfigSelector';
 
 /**
  * @typedef ErrorInfo
- * @prop {string} message
+ * @prop {string} description
```

Comment:
This nomenclature change aligns better with the `ErrorModal` prop and simplifies the usage.